compute/admin_guide/waas/deploy_waas/deployment_vpc_mirroring.adoc

```diff
@@ -12,15 +12,15 @@ For each VPC traffic mirroring rule, a VPC Observer will be created in a VPC mac
 
 === Prerequisites
 
-* Deployed Observers should have connectivity to the Prisma Cloud Compute console.
+* Console and the Observers must be running the 22.06 version or later.
```

Comment:
we need to talk about why we are mentioning 22.06 as  a requirement? This book is for 22.12 (PCCE and PCEE).